lib/modules/deployment/contract_deployer.js

```diff
@@ -140,13 +140,12 @@ class ContractDeployer {
           return next();
         }
 
-        // TODO find a better way to do that
-        if (process.env.isTest) {
-          return self.deployContract(contract, next);
-        }
-        // TODO: this should be a plugin API instead, if not existing, it should by default deploy the contract
-        self.events.request("deploy:contract:shouldDeploy", contract, function(trackedContract) {
-          if (!trackedContract) {
+        self.plugins.emitAndRunActionsForEvent('deploy:contract:shouldDeploy', {contract: contract, shouldDeploy: true}, function(_err, params) {
```

Comment:
Why shouldDeploy always true?

Reply:
because that's the default behaviour, unless an action (if any even exists) changes it to false.